CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` and `Removed`.
 
 ## [Unreleased]
+### Added
+- Added a command line option to update all addons with an update without launching the GUI. Process will exit after completing.
+  - Use `ajour update` from command line
+  - `ajour --help` can be used to see all command line options
+
```

Comment:
I think we should consider adding a section to README.md as well where we can describe how Average Joe can use this awesome feature.